fairing/functions/function_shim.py

```diff
@@ -45,5 +54,7 @@ def call(serialized_fn_file):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Shim for calling a serialized function')
     parser.add_argument('--serialized_fn_file', action="store")
+    parser.add_argument('--python_version', action="store")
     args = parser.parse_args()
+    compare_version(args.python_version)
```

Comment:
we can do a try catch here but i think leaving as it is would give the same intended behavior. 
```
$ python -c "import sdsfd"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'sdsfd'
$ echo $?
1
```

Reply:
keep the traceback without catch, so user will see as below:

```
Traceback (most recent call last):
  File "test.py", line 24, in <module>
    compare_version(args.python_version)
  File "test.py", line 18, in compare_version
    + local_python_version + ' in the local environment.')
RuntimeError: The Python version 3.6 mismatches with Python 3.5 in the local environment.
```